powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthServiceImpl.java

```diff
@@ -142,6 +146,61 @@ public GetActivationListForUserResponse getActivationListForUser(GetActivationLi
         }
     }
 
+    @Override
+    @Transactional
+    public LookupActivationsResponse lookupActivations(LookupActivationsRequest request) throws GenericServiceException {
+        if (request.getUserIds().isEmpty()) {
+            logger.warn("Invalid request");
```

Comment:
We should:
- do a null-check on `request.getUserIds()`
- be a bit more chatty here to hint that the issue was in missing user IDs.

Reply:
Ok, I will fix the logging in other methods as well.